### ISSUE TITLE
Fail to install Cloud Libs on BIG-IP 15.1.1

### ISSUE THREAD

**Issue by memes** [memes/terraform-google-f5-bigip#18]
## Describe the bug

During initial boot, cloud libraries fail to install with a new JSON error response. Works on 15.1.0.4-0.0.6

## Terraform modules used

* [x] BIG-IP standalone
* [ ] BIG-IP HA
* [ ] BIG-IP CFE

## GCP environment

* [x] Standalone project (default)
* [ ] Shared VPC Host project
* [ ] Shared VPC Service project
* [ ] Integration with serverless (Cloud Functions, Cloud Run, App Engine, etc.)

Any other details we need to know?

## Reproducing the issue

How can we reproduce this behaviour? Which version of F5's product was used?

f5-bigip-15-1-1-0-0-6-payg-good-25mbps-201008055114

## Additional context

```text
Terraform v0.13.5
+ provider registry.terraform.io/hashicorp/google v3.48.0
+ provider registry.terraform.io/hashicorp/random v3.0.0
+ provider registry.terraform.io/hashicorp/template v2.2.0

Providers required by configuration:
.
├── provider[registry.terraform.io/hashicorp/google] ~> 3.48, ~> 3.48
├── module.internal_vpc
│   ├── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.vpc
│   │   └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.routes
│   │   └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   └── module.subnets
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
├── module.control_vpc
│   ├── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.vpc
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.routes
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   └── module.subnets
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
├── module.external_vpc
│   ├── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.routes
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   ├── module.subnets
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
│   └── module.vpc
│       └── provider[registry.terraform.io/hashicorp/google] < 4.0.*, >= 2.12.*
├── module.control-nat
│   ├── provider[registry.terraform.io/hashicorp/random]
│   └── provider[registry.terraform.io/hashicorp/google]
├── module.service_sa
│   ├── provider[registry.terraform.io/hashicorp/google]
│   └── provider[registry.terraform.io/hashicorp/template]
├── module.bigip
│   ├── provider[registry.terraform.io/hashicorp/google] >= 3.48.*
│   └── module.metadata
├── module.bigip_password
│   ├── provider[registry.terraform.io/hashicorp/google] ~> 3.44
│   ├── provider[registry.terraform.io/hashicorp/random]
│   └── module.secret
│       └── provider[registry.terraform.io/hashicorp/google] ~> 3.44
├── module.bigip_sa
│   ├── provider[registry.terraform.io/hashicorp/google]
│   └── provider[registry.terraform.io/hashicorp/template]
└── module.bastion
    ├── provider[registry.terraform.io/hashicorp/google]
    ├── provider[registry.terraform.io/hashicorp/random]
    ├── module.iap_tunneling
        └── provider[registry.terraform.io/hashicorp/google]
    └── module.instance_template
        └── provider[registry.terraform.io/hashicorp/google] >= 2.7.*, < 4.0.*

Providers required by state:

    provider[registry.terraform.io/hashicorp/google]

    provider[registry.terraform.io/hashicorp/random]

    provider[registry.terraform.io/hashicorp/template]
```

*DO NOT UPLOAD OR LINK TO FILES WITH SENSITIVE INFORMATION, SERVICE ACCOUNT KEYS, ETC.*

**Comment by memes** [comment]
Confirmed same behaviour with f5-bigip-15-1-2-0-0-9-payg-good-5gbps-201110225418

**Comment by memes** [comment]
Root cause is a fix in BIG-IP v15.1.1 and v15.1.2:
> [912969-2](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/releasenotes/related/relnote-supplement-bigip-15-1-1.html#A912969-2) : No validation for input RPM source paths for package-management-tasks installation

Fix is to download RPMs to `/var/config/rest/downloads/` and install from that path.
